blocks/callout/callout.css

```diff
@@ -82,16 +105,15 @@
     font-weight: var(--spectrum-extra-bold-font-weight);
     font-size: var(--spectrum-font-size-700);
     line-height: 1.25;
-    margin-block-end: 1rem;
+    margin-block-end: 0.5rem;
 
     @container callout-container (min-width: 35.5rem) {
       font-size: var(--spectrum-font-size-900);
-      margin-block-end: 1.5rem;
     }
 
     @container callout-container (min-width: 44rem) {
       font-size: var(--spectrum-font-size-1000);
-      margin-block-end: 1rem;
+      margin-block-end: 0.75rem;
```

Comment:
🐄 👩

Reply:
0.5 for small that we talked about, then I cowgirled 0.75 at medium so it wasn't so harsh. 

i'll change it to the gap!!